posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -1831,6 +1929,43 @@
   GROUP BY `$session_id`, team_id
   
   
+  '''
+# ---
+# name: test_create_table_query[sharded_app_metrics2]
+  '''
+  
+  CREATE TABLE IF NOT EXISTS sharded_app_metrics2 ON CLUSTER 'posthog'
+  (
+      team_id Int64,
+      timestamp DateTime64(6, 'UTC'),
+      -- The name of the service or product that generated the metrics.
+      -- Examples: plugins, hog
+      app_source LowCardinality(String),
+      -- An id for the app source.
+      -- Set app_source to avoid collision with ids from other app sources if the id generation is not safe.
+      -- Examples: A plugin id, a hog application id
+      app_source_id String,
+      -- A secondary id e.g. for the instance of app_source that generated this metric.
+      -- This may be ommitted if app_source is a singleton.
+      -- Examples: A plugin config id, a hod application config id
+      instance_id String,
+      successes SimpleAggregateFunction(sum, Int64),
+      skipped SimpleAggregateFunction(sum, Int64),
+      failures SimpleAggregateFunction(sum, Int64),
+      -- The UUID of the event that caused this error to be generated.
+      error_uuid UUID,
+      error_type String,
+      error_details String CODEC(ZSTD(3))
```

Comment:
These columns have always felt clunky to me and I think that they're better suited in the new `logs` table instead of as colums in an AggregatingMergeTree table. WDYT?

Reply:
Brett and I also thought they aren't the best here, though didn't have a good solution for it yesterday, so `logs` table 🤔 
I'd keep the `error_type` for which we'd want to aggregate.
For `error_uuid` (i.e. event UUID and we should rename it) and `error_details` using the logs table seems like a good idea - to view those one anyway clicks on something that then shows a modal with the info, so at that point we could make another API call to show the info, which could come from the logs table. Here's an example page where you can see usage: https://us.posthog.com/project/2/pipeline/destinations/457/metrics